micromanager_gui/main_window.py

```diff
@@ -168,7 +168,7 @@ def get_event(viewer, event):
             if not self._mmc.getPixelSizeUm() > 0:
                 x = None
                 y = None
-                raise ValueError ('PIXEL SIZE NOT SET.')
+                raise Warning ('PIXEL SIZE NOT SET.')
```

Comment:
don't `raise Warning`.

If your program _can_ continue, but you'd like the user to know something unexpected happened, then use `import warnings;  warnings.warn("some message")`

If you _cannot_ continue the execution then use `raise`, but try to find the best corresponding [exception type](https://docs.python.org/3/library/exceptions.html#base-classes)